Android: RN 0.73: Build fails to compile C++ source

RN 0.73 fails to build the Android app when including package react-native-quick-base64. This could be an issue with the React Native library itself. Their bug reporting page says to notify the third-party library developer first.

**Steps to reproduce**
_npx react-native init RN073 --version 0.73.0_
_cd RN073_
_yarn add react-native-quick-base64_
_cd android_
_./gradlew assemble_

**Error**
Execution failed for task ':react-native-quick-base64:buildCMakeDebug[arm64-v8a]'.
`[1/5] Building CXX object CMakeFiles/quickbase64.dir/cpp-adapter.cpp.o`
`  FAILED: CMakeFiles/quickbase64.dir/cpp-adapter.cpp.o`

`In file included from RN073/node_modules/react-native-quick-base64/cpp/react-native-quick-base64.cpp:1:`
`In file included from RN073/node_modules/react-native-quick-base64/android/../cpp/react-native-quick-base64.h:1:`
`In file included from RN073/node_modules/react-native/ReactCommon/jsi/jsi/jsilib.h:10:`
`  RN073/node_modules/react-native/ReactCommon/jsi/jsi/jsi.h:1107:23: error: no template named 'enable_if_t' in namespace 'std'; did you mean 'enable_if'?`
`      typename = std::enable_if_t<`
`                 ~~~~~^~~~~~~~~~~`
`                      enable_if`

`Library/Android/sdk/ndk/25.1.8937393/toolchains/llvm/prebuilt/darwin-x86_64/sysroot/usr/include/c++/v1/type_traits:538:63: note: 'enable_if' declared here`
`  template <bool, class _Tp = void> struct _LIBCPP_TEMPLATE_VIS enable_if {};`

--- a/android/CMakeLists.txt
+++ b/android/CMakeLists.txt
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.9.0)
 
 set (PACKAGE_NAME "react-native-quick-base64")
 set (CMAKE_VERBOSE_MAKEFILE ON)
-set (CMAKE_CXX_STANDARD 11)
+set (CMAKE_CXX_STANDARD 14)
 
 file(TO_CMAKE_PATH "${NODE_MODULES_DIR}/react-native/ReactCommon/jsi/jsi/jsi.cpp" libPath)
 

